vertical-pod-autoscaler/pkg/recommender/model/aggregate_container_state.go

```diff
@@ -97,6 +97,7 @@ type AggregateContainerState struct {
 	LastRecommendation corev1.ResourceList
 	IsUnderVPA         bool
 	UpdateMode         *vpa_types.UpdateMode
+	scalingMode        *vpa_types.ContainerScalingMode
```

Comment:
Why is this one private while others are exported?

Reply:
Done